platform/ios/src/MGLMapView.mm

```diff
@@ -2139,6 +2142,9 @@ - (void)handleTwoFingerDragGesture:(UIPanGestureRecognizer *)twoFingerDrag
     
     if (twoFingerDrag.state == UIGestureRecognizerStateBegan)
     {
+        CGPoint midPoint = [twoFingerDrag translationInView:twoFingerDrag.view];
+        // Creates an angle at the beginning of the gesture
+        self.dragGestureMiddlePoint = CGPointMake(midPoint.x, midPoint.y-1);
```

Comment:
Can you elaborate on what the `- 1` represents here? Why isn’t `midPoint` itself here sufficient?

Reply:
If we start at lets say `50,50` subtracting `-1` an initial `dragGestureMiddlePoint` will be `50, 49` then when the code continues to calculate `middlePoint` (`50, 50` too) in
https://github.com/mapbox/mapbox-gl-native/pull/15349/files#diff-20502b6bc02986a4398d0da84b2d4b94R2164 will create a 90º angle because at this point and for the first execution `dragGestureMiddlePoint` is equals to `middlePoint` and would cause a slightly gesture delay

Reply:
Hrm, thanks — could you perhaps capture that in an inline comment?